generators/server/templates/src/test/resources/testcontainers.properties.ejs

```diff
@@ -0,0 +1 @@
+testcontainers.reuse.enable=true
```

Comment:
From what I've seem in docs, this won't work here.
It needs to be at the user home.

Reply:
See https://www.testcontainers.org/features/configuration/

Reply:
This behavior should be documented.
I would prefer to disable by default.

As a jhipster developer I expect that my environment in left as it was, without a running container.

Is quite common to create a project, test and delete the project.

Reply:
It is left like it was: this configuration bit just reuses the container among test case, but stop it after the whole test suite has been executed.